src/Menu/Menu.jsx

```diff
@@ -199,7 +199,7 @@ class Menu extends React.Component {
   renderTrigger(node) {
     return React.cloneElement(node, {
       onClick: this.onTriggerClick,
-      'aria-haspopup': true,
+      'aria-haspopup': 'menu',
```

Comment:
This.  This was my change, as per feedback from @wittjeff.

Reply:
No good deed goes unpunished.